platforms/jvm/language-java/src/main/java/org/gradle/external/javadoc/OptionLessJavadocOptionFileOption.java

```diff
@@ -26,6 +27,7 @@
  *
  * @param <T> The type which this option represents.
  */
+@NotToBeMigratedToLazy
```

Comment:
❓ In this case shouldn't we mark `getValue()` with `@NotToBeMigratedToLazyProperty` instead?

Reply:
We can also do it that way, yes